src/targets/gpu/prefuse_ops.cpp

```diff
@@ -162,36 +162,53 @@ auto is_mlir_gemm()
     });
 }
 
+auto is_test_gemm(bool enable_attention)
+{
+    return match::make_basic_pred_matcher([=](instruction_ref ins) {
+        if(ins->name() != "dot")
+            return false;
+        return enable_attention;
+    });
+}
+
 struct find_gemm_softmax_gemm
 {
+    bool enable_attention;
```

Comment:
Set this to `false`, so its not uninitialized when default constructing.

Reply:
done